docs/core/diagnostics/dotnet-counters.md

```diff
@@ -273,6 +273,96 @@ dotnet-counters monitor [-h|--help] [-p|--process-id] [-n|--name] [--diagnostic-
       request                                      100
   ```
 
+- View all well-known counters that are available in `dotnet-counters`:
+
+  ```console
+  > dotnet-counters list
```

Comment:
I guess I am surprised the list takes a default or runtime version at all.

I would have guess the list would be exhaustive with the applicable version in the output.